src/common/com/intellij/plugins/haxe/model/HaxeMethodModel.java

```diff
@@ -163,5 +165,26 @@ public HaxeMethodModel getParentMethod() {
   public String toString() {
     return "HaxeMethodModel(" + this.getName() + ", " + this.getParameters() + ")";
   }
+
+  @Override
+  public List<HaxeModel> getExposedMembers() {
+    return new ArrayList<>(getParameters());
```

Comment:
So, why this?  In every other case that I've seen you use "exposed," it's been for publicly addressable/exported entities.  Here, that's not the case.

Reply:
Ah, it mustn't be here.